Assets/Scripts/Actor Components/Buff.cs

```diff
@@ -5,6 +5,11 @@
 
 public class Buff : MonoBehaviour
 {
+    private readonly float timerDuration = 3f;
```

Comment:
Consider making `timerDuration` a normal serialized field so that we can edit it in the inspector.